about/why-r.md

```diff
@@ -0,0 +1,31 @@
+---
+layout: page
+title: Why R?
+---
+
+I chose to add an R version of this course for a handful of reasons
+including:
+
+1.  R is used widely in a number of disciplines, like ecology, and provides a  
+    solid platform for beginner scientific programmers. Many graphing and 
+    statistical operations are built right into the language.
+2.  It is a broadly useful language. This means that you should be able to do 
+    most anything you want to do using R, and it should be relatively easy to 
+    accomplish.
+3.  It's free. Funding rates for scientific research are extremely low
+    at the moment and this means that expensive software licences are
+    often untenable, especially for students conducting their own
+    research. This also allows students in the class to program on any
+    computer they want without dealing with licensing hassles, and
+    guarantees that anyone can take advantage of the online resources
+    that are being developed.
+4.  It runs on all major operating systems.
+5.  It is used by many scientific programmers and taught in many research
+    science departments. This makes collaborating with experts
+    on large or complex projects much easier.
+6.  [Data Carpentry](http://datacarpentry.org), a truly
+    excellent resource for learning scientific programming, is available
```

Comment:
Since this is going to be a Data Carpentry product we should probably pull "a truly excellent resource..." as it might come off a bit oddly.